### PR TITLE
Fix transaction validation for empty commands

### PR DESCRIPTION
**Pull request by luckychess**
## What is this pull request?
Fix for empty transaction validation
   
## Why do you implement it? Why do we need this pull request?
Transaction without commands are not allowed
  
## How to use the features provided in the pull request?
Test included

## Details/Features
- Add validation step for zero commands in transaction case
- Add test for empty transaction validation

### CODE REVIEW

File: shared_model/validators/commands_validator.hpp [hunk 1]
```diff
@@ -90,7 +90,8 @@ namespace shared_model {
 
         ReasonsGroupType operator()(
             const detail::PolymorphicWrapper<interface::CreateAccount> &ca)
-            const {;
+            const {
+          ;
```
Comment (l4l) [inline shared_model/validators/commands_validator.hpp]: Remove it plz D:

Comment (luckychess) [reply shared_model/validators/commands_validator.hpp]: Ooops )

File: test/module/shared_model/validators/commands_validator_test.cpp [hunk 2]
```diff
@@ -193,6 +193,20 @@ iroha::protocol::RevokePermission generateRevokePermission(
...
+ * @then answer has error about empty transaction
+ */
+TEST(commandsValidatorTest, EmptyTransactionTest) {
+  auto tx = generateEmptyTransaction();
+  tx.mutable_payload()->set_created_time(iroha::time::now());
+  shared_model::validation::CommandsValidator commands_validator;
+  auto answer = commands_validator.validate(
+          detail::make_polymorphic<proto::Transaction>(tx));
+  ASSERT_TRUE(answer.hasErrors());
+}
```
Comment (kamilsa) [inline test/module/shared_model/validators/commands_validator_test.cpp]: We can also check how many errors did we get from answer. Like here https://github.com/hyperledger/iroha/pull/768/files#diff-c76782c8a0c46a09b1c21cbbe5dccb3bR332
